typescript/sdk/src/hook/EvmHookReader.ts

```diff
@@ -156,11 +164,7 @@ export class EvmHookReader implements HookReader {
     const hookConfigs: DerivedHookConfig[] = await concurrentMap(
       this.concurrency,
       hooks,
-      async (hook) => {
-        const hookConfig = await this.deriveHookConfig(hook);
-        assert(hookConfig, `No hook config found for ${hook}.`);
-        return hookConfig;
-      },
+      this.deriveHookConfig,
```

Comment:
```suggestion
      async (hook) => this.deriveHookConfig(hook),
```
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3543#discussion_r1550568532

Reply:
for posterity (courtesy of @pbio): https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4062#discussion_r1657336654